test/helpers.js

```diff
@@ -254,14 +254,18 @@ describe('helpers', function () {
     })
 
     describe('parameters', function () {
-      it('should not overwrite parameters from path when exists in operation', function () {
+      it('should add parameters from path to operation without overwriting', function () {
```

Comment:
@webron is this the desired behaviour?
Should global `parameters` get merged with operation-level `parameters`? 
Just want to clarify this point... since we've done both in the past.